.github/workflows/build-image-pr.yml

```diff
@@ -37,18 +36,24 @@ jobs:
         uses: helm/kind-action@v1.9.0
       - name: Load Local Registry Test Image
         env:
-          IMG: "${{ env.IMG }}:${{ steps.tags.outputs.tag }}"
+          IMG: "${{ env.IMG_ORG }}/${{ env.IMG_REPO }}:${{ steps.tags.outputs.tag }}"
         run: |
           kind load docker-image -n chart-testing ${IMG}
       - name: Create Test Registry
         env:
-          IMG: "${{ env.IMG }}:${{ steps.tags.outputs.tag }}"
+          IMG: "${{ env.IMG_ORG }}/${{ env.IMG_REPO }}:${{ steps.tags.outputs.tag }}"
         run: |
+          echo "Download kustomize 5.2.1"
+          mkdir $GITHUB_WORKSPACE/kustomize
+          curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash -s "5.2.1" "$GITHUB_WORKSPACE/kustomize"
```

Comment:
Same question as other PR. Can we try to avoid using curl | bash?

Reply:
I tried first an alternative with

https://github.com/kubeflow/model-registry/pull/43/commits/471563bb1484725afa4ed83e1f3e74fb929dc6a8#diff-ee244f6eeb2d7ac8c604cdc91588b49204dea5594df11b20af816ad9f4b54467R36-R39

that is using a dedicated GHA to download kustomize, but the performance was horrible due to the fact the GHA action step does much more we don't currently need.

So at this time, this seems the most vanilla and default way to override for a specific version of kustomize that what bundled in GHA automatically

Reply:
btw, by "vanilla and default way" I mean what indicated by the manual, here: https://kubectl.docs.kubernetes.io/installation/kustomize/binaries/